packages/core/examples/colorbar/index.ts

```diff
@@ -0,0 +1,728 @@
+// This is for debugging purposes
+console.warn(
+  'Click on index.ts to open source code for this example --------->'
+);
+
+const {
+  PanTool,
+  WindowLevelTool,
+  StackScrollMouseWheelTool,
+  ZoomTool,
+  ToolGroupManager,
+  Enums: csToolsEnums,
+} = cornerstoneTools;
+
+const { MouseBindings } = csToolsEnums;
+
+const pause = (interval) =>
+  new Promise((resolve) => setTimeout(resolve, interval));
+
+const { ViewportType } = Enums;
+const renderingEngineId = 'myRenderingEngine';
+const stackViewportId = 'CT_STACK';
+const volumeViewportId = 'CT_VOLUME_SAGITTAL';
+const volumeToolGroupId = 'VOLUME_TOOL_GROUP_ID';
+
+// Define unique ids for the volumes
+const volumeLoaderScheme = 'cornerstoneStreamingImageVolume'; // Loader id which defines which volume loader to use
+const ctVolumeName = 'CT_VOLUME_ID'; // Id of the volume less loader prefix
+const ctVolumeId = `${volumeLoaderScheme}:${ctVolumeName}`; // VolumeId with loader id + volume id
+
+// Define a unique id for the volume
+const ptVolumeName = 'PT_VOLUME_ID';
+const ptVolumeId = `${volumeLoaderScheme}:${ptVolumeName}`;
+
+const colormaps = vtkColormaps.rgbPresetNames.map(
+  (presetName) => vtkColormaps.getPresetByName(presetName) as Colormap
+);
+let currentPTColormapName = 'Black-Body Radiation';
+let ctColorBar = null;
+let ptColorBar = null;
+const voiRangeMin = 0;
+const voiRangeMax = 1;
+
+// ==[ Set up page ]============================================================
+
+setTitleAndDescription(
+  'Volume Viewport API With Multiple Volumes',
+  'Demonstrates how to interact with a Volume viewport when using fusion.'
+);
+
+const content = document.getElementById('content');
+const element = document.createElement('div');
+element.id = 'cornerstone-element';
+
+Object.assign(element.style, {
+  position: 'relative',
+  width: '500px',
+  height: '500px',
+  marginBottom: '30px',
+});
+
+content.appendChild(element);
+
+const rightTopContainer = document.createElement('div');
+const rightBottomContainer = document.createElement('div');
+const bottomLeftContainer = document.createElement('div');
+const bottomRightContainer = document.createElement('div');
+
+const containers = [
+  rightTopContainer,
+  rightBottomContainer,
+  bottomLeftContainer,
+  bottomRightContainer,
+];
+
+const info = document.createElement('div');
+content.appendChild(info);
+
+const addInstruction = (instruction) => {
+  const node = document.createElement('p');
+  node.innerText = instruction;
+  info.appendChild(node);
+};
+
+addInstruction('- Select different colormaps');
+addInstruction('- Click and drag on viewport to change VOI');
+addInstruction('- Click and drag at the color bar to change VOI');
+
+const colorBarSize = { shortSide: 20, longSide: 250 };
+
+// const ctColorBar = new ViewportColorBar({
+//   id: 'ctColorBar',
+//   viewportId: volumeViewportId,
+//   renderingEngineId,
+//   colormaps,
+//   activeColormapName: 'Grayscale',
+//   // voiRange: { min: 0.25, max: 0.75 },
+// });
```

Comment:
Upon reviewing the props, there is no need for the renderingEngineId since it can be obtained from the viewport. Moreover, it is preferable that this component is not associated with the range or voiRange. Ideally, all of these functionalities should be seamlessly integrated behind the scenes.